packages/account/tests/account.e2e.spec.ts

```diff
@@ -104,14 +106,59 @@ describe("Account Tests", () => {
     expect(newBalance).toEqual(balance);
   }, 60000);
 
+  it("Should gaslessly mint an NFT on Base base", async () => {
```

Comment:
No need to do this test, we should use Base testnet only in Multichain test, if NFT mint works on mumbai there is not need to test on other testnets, it will just drain more test funds from the wallets.

Reply:
removed